packages/medusa/src/repositories/customer-group.ts

```diff
@@ -1,5 +1,64 @@
-import { EntityRepository, Repository } from "typeorm"
+import {
+  EntityRepository,
+  Repository,
+  getManager,
+  getConnection,
+} from "typeorm"
+import { MedusaError } from "medusa-core-utils"
 import { CustomerGroup } from "../models/customer-group"
+import { CustomerGroupsBatchCustomer } from "../types/customer-groups"
+import { Customer } from ".."
 
 @EntityRepository(CustomerGroup)
-export class CustomerGroupRepository extends Repository<CustomerGroup> {}
+export class CustomerGroupRepository extends Repository<CustomerGroup> {
+  async addCustomerBatch(
+    groupId: string,
+    customerIds: CustomerGroupsBatchCustomer[]
```

Comment:
I would change this function to take `customerIds` here as strings and let `CustomerGroupsBatchCustomer` remain an API type only.